apps/etterlatte-saksbehandling-ui/.nais/prod.yaml

```diff
@@ -86,6 +86,8 @@ spec:
       value: api://prod-gcp.etterlatte.etterlatte-trygdetid/.default
     - name: GOSYS_URL
       value: https://gosys.intern.nav.no/gosys
+    - name: EESSI_PENSJON_URL
+      value: https://pensjon-utland.intern.dev.nav.no/gjenny
```

Comment:
Er det riktig med `.dev` i url-en her for prod?

Reply:
Absolutt ikke... 🤦‍♂️ Korrigert nå!